client/rest/src/plugins/rosetta/OperationParser.js

```diff
@@ -161,15 +181,17 @@ export class OperationParser {
 				// due to async nature, operations from previous transaction might not be added yet
 				// so simply using operations.length would yield wrong result
 				// instead, renumber the operations below
-				const subOperations = await this.parseTransactionInternal(0, subTransaction.transaction);
+				const subOperations = await this.parseTransactionInternal(
+					subTransaction.transaction,
+					subTransaction.meta && undefined !== subTransaction.meta.index
```

Comment:
Just wondering when you will not be present. 🤔

Reply:
when JSON is derived from sdk model